pull_requests.rb

```diff
@@ -58,27 +58,34 @@
 
 parser.parse!
 
-options[:file] = 'modules.json' if options[:file].nil?
+options = {}
+options[:oauth] = ENV['GITHUB_COMMUNITY_TOKEN'] if ENV['GITHUB_COMMUNITY_TOKEN']
+parser = OptionParser.new do |opts|
+  opts.banner = 'Usage: stats.rb [options]'
```

Comment:
```suggestion
  opts.banner = 'Usage: pull_requests.rb [options]'
```

please also double check the other scripts